src/components/Input/Input.tsx

```diff
@@ -127,6 +145,18 @@ export const Input = (
     addonAfterProp && renderAddon('after', addonAfterProp)
   ), [addonAfterProp, renderAddon])
 
+  const getValue = useCallback((strOrObj?: string | Record<string, unknown>) => {
+    return typeof strOrObj === 'object' ? String(strOrObj[valuePropName]) : strOrObj
```

Comment:
Is too much fragile -> 
<img width="774" alt="Screenshot 2024-11-26 alle 10 19 10" src="https://github.com/user-attachments/assets/c7504f7b-7623-40d7-9b7e-c31d966603df">

Reply:
we've added a more robust check with the function  isObject here https://github.com/mia-platform/design-system/blob/feat/input-addon-custom-value-names/src/utils/object.ts